Hearthstone Deck Tracker/Windows/MainWindow/MainWindow_Edit.cs

```diff
@@ -317,6 +317,13 @@ internal void BtnEditDeck_Click(object sender, RoutedEventArgs e)
 			TagControlEdit.SetSelectedTags(deck.Tags);
 		}
 
+		internal async void BtnSetDeckUrl_Click(object sender, RoutedEventArgs e)
+		{
+			var selectedDeck = DeckPickerList.SelectedDecks.FirstOrDefault();
+			selectedDeck.Url = await InputDeckURL();
```

Comment:
One more thing: selectedDeck *could* in theory be null here. I think in reality there is no way for this method to be called without a deck being selected, but `FirstOrDefault()` should be follow by a null check nonetheless.